hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryMetrics.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.hedera.node.app.workflows.query;
+
+import static com.hedera.node.app.spi.HapiUtils.QUERY_FUNCTIONS;
+
+import com.hedera.hapi.node.base.HederaFunctionality;
+import com.swirlds.common.metrics.Counter;
+import com.swirlds.common.metrics.Metrics;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.EnumMap;
+import java.util.Map;
+import javax.inject.Inject;
+
+/**
+ * A container for metrics related to queries.
+ */
+public class QueryMetrics {
```

Comment:
Actually, I propose we eliminate these metrics entirely. They are already covered by the QueryMethod class in gRPC.

Reply:
If both cases are handled (received and answered) I am fine with that.

Reply:
Done